gradle/versions.gradle

```diff
@@ -11,7 +11,7 @@ ext.libVersions = [
     jackson: '2.5.1',
     jackson_annotation: '2.5.0',
     cassandra: '2.2.1',
-    cassandra_driver_core: '2.2.0-rc3',
+    cassandra_driver_core: '2.1.5',
```

Comment:
This issue https://datastax-oss.atlassian.net/browse/JAVA-823 and the amount of CQL tests failing on pg right now are the main reason.

This is a hopefully temporary fix to limit the issues with the current driver until it's fixed in the future. We are aware of the fact that this will bring many unshaded dependency, and we will use a shaded version of the driver in pg.